snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -198,25 +189,20 @@
         downloadUserCountry();
     }
 
-    {# In preview mode, ignore geolocation. #}
     {% if preview %}
     function chooseSnippet(snippets) {
-        return snippets[Math.floor(Math.random() * snippets.length)];
+        return snippets[0];
```

Comment:
Will this not always return the same snippet from the array or, are we popping items of the array as well somewhere? Perhaps I am just reading it wrong?